src/transformers/generation/continuous_batching/continuous_api.py

```diff
@@ -561,27 +555,39 @@ def _sync(self) -> list[int]:
     @traced
     def _maybe_send_output(self, state: RequestState) -> None:
         """Send output to the queue based on streaming mode and request state."""
-        if state.streaming:
-            self.output_queue.put(state.to_generation_output())
-        elif state.status == RequestStatus.FINISHED:
+        if state.streaming or state.status == RequestStatus.FINISHED:
             self.output_queue.put(state.to_generation_output())
```

Comment:
In that case, wdyt of this:
```suggestion
        send_output = state.streaming or state.status == RequestStatus.FINISHED
        if not send_output:
            return
        self.output_queue.put(state.to_generation_output())
```
imo reads better